app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -81,9 +84,11 @@ class EditProfileFragment : Fragment() {
             val selectedImage = BitmapFactory.decodeStream(imageStream)
             encodedImage = encodeImage(selectedImage)
 
+
+
             Picasso.get()
                     .load(imageUri)
-                    .placeholder(R.drawable.ic_person_black_24dp)
+                    .placeholder(AppCompatResources.getDrawable(context as Context, R.drawable.ic_person_black_24dp) as Drawable)
```

Comment:
`context as Context` to `context

Reply:
Same as no use of `as Drawable`

Reply:
Thanks, Areeb. 
Sorry, did not understand how to change. 

Do you mean I can use backquote instead of the cast? Did not find any explanation of such kind; sorry, I’m new to Kotlin.

Reply:
Just remove `as Context` and `as Drawable`

Reply:
I started with it, but android studio makes it red and says:  method expects type Context and in fact gets Context?  (Or maybe vice versa - forgot). And similarly about Drawable.

Reply:
Ok. Will do so.